internal/command/test_test.go

```diff
@@ -780,58 +780,98 @@ can remove the provider configuration again.
 	}
 }
 
-func TestTest_NestedSetupModules(t *testing.T) {
-	td := t.TempDir()
-	testCopyDir(t, testFixturePath(path.Join("test", "with_nested_setup_modules")), td)
-	defer testChdir(t, td)()
+func TestTest_Modules(t *testing.T) {
```

Comment:
I merged a few similar tests together - `TestTest_OnlyExternalModules`, `TestTest_NestedSetupModules` and the new `with_verify_module`

Reply:
Do I understand correctly that this new test is a regression test that fails before the change?

Reply:
Indeed 🙃